daft/dataframe/dataframe.py

```diff
@@ -1277,6 +1278,66 @@ def from_ray_dataset(cls, ds: "RayDataset") -> "DataFrame":
             )
         )
 
+    @DataframePublicAPI
+    def to_dask_dataframe(self) -> "dask.DataFrame":
+        """Converts the current Daft DataFrame to a Dask DataFrame.
+
+        The returned Dask DataFrame will use `Dask-on-Ray <https://docs.ray.io/en/latest/data/dask-on-ray.html>`__
+        to execute operations on a Ray cluster.
+
+        .. NOTE::
+            This function can only work if Daft is running using the RayRunner.
+
+        Returns:
+            dask.DataFrame: A Dask DataFrame stored on a Ray cluster.
+        """
+        from daft.runners.ray_runner import RayPartitionSet
+
+        self.collect()
+        partition_set = self._result
+        assert partition_set is not None
+        # TODO(Clark): Support Dask DataFrame conversion for the local runner if
```

Comment:
That assumption that a user wants to use Dask-on-Ray if they're already using Daft with the Ray runner sounds good to me! We would need to set the scheduler config globally for the user, but I think that should be fine.

>Otherwise, we could also pass-through the scheduler as a kwarg

Note that the "scheduler as kwarg" is passed at Dask execution time, which we won't have control over since we're returning an unexecuted Dask DataFrame to the user, so our only hook for setting the scheduler for the user is the global config.

Reply:
I see - that makes sense! In other words, we wouldn't control when the user calls `.compute()`.

I think I misunderstood the model for working with Dask+Ray - PR should be good as is, we probably would want to avoid side-effects like setting the global scheduler config implicitly from within our function 😬